package/boot/uboot-envtools/files/rockchip

```diff
@@ -0,0 +1,23 @@
+#

```

Comment:
this file needs to convert to use LF line breaks, and add new line at the end of file.
would be better to name as `rockchip_armv8`.

otherwise lgtm.